OPLS.py

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env python
+### Author:
+# Daisy Y. Kyu
+# Nam Thi
+# Victoria Lim (limvt@uci.edu)
+# Caitlin C. Bannan (bannanc@uci.edu)
+
+### Description: This Python script minimize a mol2 files 
+#     with specific forcefield type and location provided.
+#     For N input *.mol2 files, there should be N output
+#     *.mol2 files, unless the input file is missing
+
+
+### Example usuage:
+# - python OPLS.py --ifile /input_directory/mol2 --opttype fftype > output.dat
+# - python OPLS.py -i /input_directory/OPLS2005/mol2 -o OPLS2005  > output.dat
+
+### Dependencies: 
+# - For all: needs fftype and location of mol2 files. (-i, -o)
+# - cmd command: refer to the help message in utilities on 
+#   specifying structures from other file formats.
+
+
+### Supported force field:
+# - OPLS3
+# - OPLS2005
+
+import os
+
+  # ---------------------------- Functions ------------------------- #
+
+def OPLSMin(ifile,opttype):
+    """
+   
+    Take one mol2 file and do the minimization, then output into
+    a specific location
+   
+    Parameters:
+    ifile: input file directory
+    opttpe: force field type
+    Note: the output directory is specified inside the command line.
+          Refer to the help message in utilities for more information.
+    
+   
+    """
+
+    if opttype == 'OPLS3':  #specify which fftype to be run
+
+        cmd = '$SCHRODINGER/utilities/ffld_server  -version 16 -charges_from_ct -virt -no_cm1a_bcc -opt -BFGS -imol2 /input_location/%s -omol2 /output_location/%s' % (ifile,ifile.split('/')[-1])
```

Comment:
@nthi0625, we talked about coding practices of making very long lines shorter with `\`. Can you update this script with that, verifying that the functions still work as expected?